staging/src/k8s.io/client-go/util/jsonpath/jsonpath_test.go

```diff
@@ -280,6 +280,7 @@ func TestStructInput(t *testing.T) {
 
 	missingKeyTests := []jsonpathTest{
 		{"nonexistent field", "{.hello}", storeData, "", false},
+		{"nonexistent field 2", "before-{.hello}after", storeData, "before-after", false},
```

Comment:
how is this related? I don't see it in the reverted PR.

Reply:
this is a behaviour that got broken by the reverted commit. Ie the template got discarded after the `{.hello}` element.